client/__tests__/e2e/test_config.yaml

```diff
@@ -3,7 +3,7 @@ server:
     force_https: true
     port: 5005
     debug: true
-    verbose: true
+    verbose: false
```

Comment:
for my own education: why is this `false` now?

Reply:
It was an attempt to clean up the GHA logs for smoke tests.  The info level logs from the cellxgene server aren't necessary for the smoke tests, and this verbose config attribute _should_ be connected to that feature. I think this aspect of the config is broken though.

Reply:
ah so changing this doesn't produce less logs? If so should we just revert it back to `true` for now until we figure out whether we should even delete the config here?

Reply:
Config is definitely needed.  I can delete this line if we want since it defaults to `false`, but I think `false` is the correct value

Reply:
Oh I meant to restore the value back to `true`, since that’s what it was before right?

And since changing it to `false` is not reducing the logs, it feels like we don’t know what impact changing it to `false` actually does, so I was thinking to be safe, maybe just revert it back to `true`, so we don’t accidentally break something we don’t know

Reply:
```suggestion
    verbose: true
```